### ISSUE TITLE
low: replace effect method to some unsubscriber registrator

### ISSUE THREAD

**Issue by betula** [betula/realar#49]
```javascript
effect(() => () => ( // Ugly (((
  this.items_destroy && this.items_destroy()
));

// Must be pretty

unsub(() => {
  this.items_destroy && this.items_destroy()
});

// ... Think about

// Or perfect one!

un(() => {
  this.items_destroy && this.items_destroy()
});

```

**Comment by betula** [comment]
https://github.com/betula/realar/commit/e8b5e382e59ea56d0db0f04ac6ae92a1c09b31a3#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R498 - `un` function added

**Comment by betula** [comment]
Implemented in 0.6.0
